services/webinar/api.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
We could rename this file/module from 'api' to 'register' or similar as that seems to be what it's doing and would be more intention revealing than the current name?